docs/customer/did.md

```diff
@@ -48,7 +50,12 @@ Set the maximum number of INBOUND concurrent calls in **Channels**, and Calls Pe
 
 For more details on these fields, see [**Media in Customer Routing**](https://docs.connexcs.com/customer/routing/#media).
 
+<<<<<<< HEAD
+
++ **RTP Proxy Mode**: If relaxed is chosen and a connection through our service fails, it will automatically failover to backup.
```

Comment:
@Bani2791 If relaxed is chosen and a connection through our service fails, we will attempt to send the call out to your carrier direct. This will expose your customer IP address to your carrier and vice versa.